buildSrc/src/main/groovy/swatch.quarkus-rest-client-conventions.gradle

```
@@ -0,0 +1,52 @@
+import org.openapitools.generator.gradle.plugin.tasks.GenerateTask
+
+plugins {
+    id "java-library"
+    id "org.kordamp.gradle.jandex"
+    id "org.openapi.generator"
+    id "swatch.java-conventions"
+}
+
+ext {
+    api_spec_path = "${projectDir}/${project.name.replace('-client', '-api-spec.yaml')}"
+    java_package = "com.redhat.swatch.clients.${project.name.replace('-client', '').replaceAll('-', '.')}"
+}
+
+task generateApiDocs(type: GenerateTask) {
+    generatorName = "html"
+    inputSpec = api_spec_path
+    outputDir = "$buildDir/docs"
+    generateApiDocumentation = true
+    generateModelDocumentation = true
+    generateModelTests = false
+    generateApiTests = false
+    withXml = false
+}
+
+openApiGenerate {
```

So in line with what I was going for in #874 
```suggestion
tasks.register("apiGenerate", GenerateTask) {
```
That keeps it so the task to generate the API is "apiGenerate" everywhere instead of "openApiGenerate" some places and "apiGenerate" in others ("apiGenerate" is the name I had to use in the main project since we are doing 2 generations for each internal API spec)